NitroxClient/GameLogic/PlayerModelBuilder/ColorValueRange.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace NitroxClient.GameLogic.PlayerModelBuilder
+{
+    public class ColorValueRange
```

Comment:
Just `ValueRange`?

Reply:
Hrmm, I only *intended* for this to be a value object that is used only for the color filter - hence the name.